Tests/UnitTests/Networking/Backend/BaseBackendPostReceiptDataTestClass.swift

```diff
@@ -17,10 +17,15 @@ import XCTest
 
 @testable import RevenueCat
 
-class BackendPostReceiptDataTests: BaseBackendTests {
+class BaseBackendPostReceiptDataTestClass: BaseBackendTests {
```

Comment:
Ending the class name with "Class" feels weird.

Reply:
Yeah, agreed. I couldn't think of a better name before, but I've updated it now.